### PR TITLE
refactor(cxl-ui): cxl-section - remove top margin

### PR DESCRIPTION
**Pull request by freudFlintstone**
From discussion: https://cxlworld.slack.com/archives/C01HXUNGEPM/p1701945675624519

### CODE REVIEW

**Review by lkraav — COMMENTED** [review]
I would've thought https://github.com/conversionxl/aybolit/blob/0e1091cd6f67b5bd9ab14547f8982ebb445fe8d3/packages/cxl-ui/scss/cxl-section.scss#L64 just this needs to go away?

File: packages/cxl-ui/scss/global/cxl-section.scss [hunk 1]
```diff
@@ -19,10 +21,6 @@ cxl-section#view-hero, cxl-section[theme~="cxl-hero"] {
   }
 
   &.wide-background-light-blue {
-    #view-hero .cxl-hero-titles {
```
Comment (freudFlintstone) [inline packages/cxl-ui/scss/global/cxl-section.scss:22]: Unused code meant originally to position the instructor name caption on course pages

Comment (freudFlintstone) [reply packages/cxl-ui/scss/global/cxl-section.scss:22]: @lkraav it's unrelated to the issue, but it's in the same file and could affect some pages

**Comment by lkraav** [comment]
> I would've thought
> 
> https://github.com/conversionxl/aybolit/blob/0e1091cd6f67b5bd9ab14547f8982ebb445fe8d3/packages/cxl-ui/scss/cxl-section.scss#L64
> just this needs to go away?

I see a :+1: emoji but no updates to PR?

**Comment by freudFlintstone** [comment]
> > I would've thought
> > https://github.com/conversionxl/aybolit/blob/0e1091cd6f67b5bd9ab14547f8982ebb445fe8d3/packages/cxl-ui/scss/cxl-section.scss#L64
> > 
> > just this needs to go away?
> 
> I see a 👍 emoji but no updates to PR?

@lkraav, done. Got distracted and forgot.

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**